test/binder/binder_test.cpp

```diff
@@ -818,4 +819,121 @@ TEST_F(BinderCorrectnessTest, BindDepthTest) {
   EXPECT_EQ(1, exists_sub_expr_select_ele->GetDepth());
 }
 
+// NOLINTNEXTLINE
+TEST_F(BinderCorrectnessTest, CreateDatabaseTest) {
+  // Check if nested select columns are correctly processed
+  BINDER_LOG_DEBUG("Checking create database.");
+
+  std::string create_sql = "CREATE DATABASE C;";
+  auto parse_tree = parser_.BuildParseTree(create_sql);
+  auto statement = parse_tree.GetStatements()[0];
+  EXPECT_NO_THROW(binder_->BindNameToNode(statement, &parse_tree));
+}
+
+// NOLINTNEXTLINE
+TEST_F(BinderCorrectnessTest, CreateTableTest) {
```

Comment:
How do we handle create table with a primary key?

Reply:
I think the binder nor the transformer need to explicitly handle that. 

Primary key info is handled by parser. CreateStatement that corresponds to a create table query has vector of ColumnDefinitions that each embeds a `is_primary_` attribute. I'm letting Binder to not unwrap the columnDefinition until the query is transformed to a physical operator.

Reply:
> How do we handle create table with a primary key?



I think the binder nor the transformer need to explicitly handle that.

Primary key info is handled by parser. CreateStatement that corresponds to a create table query has vector of ColumnDefinitions that each embeds a is_primary_ attribute. I'm letting Binder to not unwrap the columnDefinition until the query is transformed to a physical operator.